README.md

```diff
@@ -18,6 +19,24 @@ don't always test older versions.
 If you're interested in using the [GraphQL API v4][], the recommended library is
 [shurcooL/githubv4][].
 
+## Installation ##
+
+go-github is compatible with modern Go releases in module mode, with Go installed :
+
+```bash
+go get -u github.com/google/go-github/v33
+```
+
+will resolve and add the package to the current development module, along with its dependencies.
+
+Alternatively the same can be achieved if you use import in a package:
+
+```go
+import "github.com/google/go-github/v33/github"
+```
+
+and run `go get` without paramters
+
```

Comment:
How about adding this to the end of this section so that the question never has to come up again?

```
Finally, to use the top-of-trunk version of this repo, use the following command:

` ` `bash
go get -u github.com/google/go-github/v33@master
` ` `
```

(Obviously, please fix the backticks.)